README.md

```diff
@@ -25,30 +25,54 @@
 
 <p align="center">
   <a href="https://strapi.io">
-    <img src="https://blog.strapi.io/content/images/2018/08/github_preview-2.png" />
+    <img src="https://strapi.io/assets/images/readme.png" />
   </a>
 </p>
 
 <br>
 
-Strapi is a free and open-source Headless CMS delivering your content anywhere you need.
+Strapi is a free and open source headless CMS delivering your content anywhere you need.
 
-- **Keep control over your data**. With Strapi, you know where your data is stored and you keep full control at any time.
-- **Self hosted**. You can host and scale Strapi projects the way you want. You can choose any hosting platform you want: AWS, Netlify, Heroku, a VPS or a dedicated server. You can scale as you grow, 100% independently.
-- **Database agnostic**. You can choose the database you prefer. Strapi works with SQL & NoSQL databases: MongoDB, PostgresQL, MySQL, MariaDB, SQLite.
-- **Customisable**. You can easily build your own logic by fully customising APIs, routes or plugins to perfectly fit your needs.
+- **Keep control over your data**. With Strapi, you know where your data is stored, and you keep full control at all times.
+- **Self hosted**. You can host and scale Strapi projects the way you want. You can choose any hosting platform you want: AWS, Netlify, Heroku, a VPS, or a dedicated server. You can scale as you grow, 100% independent.
+- **Database agnostic**. You can choose the database you prefer. Strapi works with SQL & NoSQL databases: MongoDB, PostgreSQL, MySQL, MariaDB, and SQLite.
+- **Customizable**. You can quickly build your logic by fully customizing APIs, routes, or plugins to fit your needs perfectly.
 
 ## Getting Started
 
 <a href="https://strapi.io/documentation/3.0.0-beta.x/getting-started/quick-start.html" target="_blank">Read the Getting Started tutorial</a> or follow the steps below:
 
-#### 🖐 Requirements
+### ⏳ Installation
+
+Install Strapi with this **Quickstart** command to install and create a Strapi project instantly:
```

Comment:
Add a note that this uses SQLite

Reply:
Done.

Reply:
What about :

Get started instantly with the quickstart command to create a Strapi project.

(So we avoid the create repetition)

Reply:
Thanks @soupette - I agree that the repetition of the word **"install"** is a bit awkward. (I think you're referring to this.) But you will see I only made the following change:

**Install Strapi with this **Quickstart** command to create a Strapi project instantly:**

As it still reads a better than rearranging the subject of the sentence (as in your example).